assignments/lambda-classes.js

```diff
@@ -1 +1,123 @@
 // CODE here for your Lambda Classes
+class Person {
+    constructor(pers) {
+        this.name = pers.name;
+        this.age = pers.age;
+        this.location = pers.location;
+    }
+    speak() {
+        return `Hello my name is ${this.name}, I am from ${this.location}`
+    }
+}
+
+class Instructor extends Person  {
+    constructor(inst) {
+        super(inst);
+        this.specialty = inst.specialty;
+        this.favLanguage = inst.favLanguage;
+        this.catchphrase = inst.catchphrase;
+    }
+    demo(subject) {
+        return `Today we are learning about ${subject}`
+    }
+    grade(student, subject) {
+        return `${student.name} receives a perfect score on ${subject}`
+    }
+    giveGrade(student) {
+        return (student.grade - Math.floor(Math.random() * student.grade));
```

Comment:
I like how you are exploring things that haven't been covered explicitly in class.